include/libdnf/rpm/package_query.hpp

```diff
@@ -587,6 +587,10 @@ class PackageQuery : public PackageSet {
     // TODO(dmach): consider removing the installed packages during the filtering
     void filter_priority();
 
+    /// Filter packages, which are installed but not available in any enabled repository.
+    /// @param with_versions If false (default) extras calculation is based only on `name.arch`. That means package is not in extras if any version of the package exists in any of the enabled repositories. If true, filter_extras is more strict and returns each package which exact NEVRA is not present in any enabled repository.
```

Comment:
What about to replace `and returns each package which exact NEVRA is not present in any repository.` by `and keeps in query installed packages which exact NEVRA is not present in any repository.`